Sources/HaCWebsiteLib/Utils/customEvent.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
What's your opinion on putting this and the test script in the repo?

I think this example is useful documentation, so maybe we do want to do it.

However, this will encourage people using the script to actually make changes, which is fine. Problem is, that they'll edit this file and I can see it accidentally getting committed and it's just another thing we have to spot in code reviews.

I think a markdown file with an example curl/postman/whatever script would prevent this. The user would have to copy and paste, then make their changes. It also allows us to document our API better.

Reply:
I think that's a good idea :)

Reply:
@Pinpickle do you think this markdown would belong in a different directory?

Reply:
Yeah maybe a `docs/` folder?